src/MycilaESPConnect.cpp

```diff
@@ -853,6 +870,10 @@ void Mycila::ESPConnect::_onWiFiEvent(WiFiEvent_t event) {
       }
       break;
 
+    case ARDUINO_EVENT_WIFI_STA_GOT_IP6:
+      LOGD(TAG, "[%s] WiFiEvent: ARDUINO_EVENT_WIFI_STA_GOT_IP6: %s", getStateName(), WiFi.globalIPv6().toString().c_str());
+      break;
```

Comment:
code is incomplete: this will only work on a mix env where ipv4 AND ipv6 is enabled otherwise the state is never switched to _setState(Mycila::ESPConnect::State::NETWORK_CONNECTED);